seq2science/envs/macs2.yaml

```diff
@@ -7,3 +7,4 @@ dependencies:
   - bioconda::macs2=2.2.7
   - bioconda::khmer=2.0
   - conda-forge::jq=1.6
+  - anaconda::setuptools=v50.3.1
```

Comment:
wait wut?

Reply:
Yeah I don't know. I got some problems on my own computer with the environment. I googled and the problem was supposed to be caused by an old setuptools. I don't see what caused the problem, but my idea was that this doesn't hurt.